main.go

```diff
@@ -64,6 +64,22 @@ func setupFlags(app *kingpin.Application) {
 	app.GetFlag("help").Short('h')
 }
 
+func cliLinterOverrides(app *kingpin.Application, element *kingpin.ParseElement, ctx *kingpin.ParseContext) error {
+	// expected input structure - <name>:<command-spec>
+	parts := strings.SplitN(*element.Value, ":", 2)
+	if len(parts) < 2 {
+		return fmt.Errorf("incorrectly formatted input: %s", *element.Value)
+	}
+	name := parts[0]
+	spec := parts[1]
+	conf, err := parseLinterSpec(name, spec)
+	if err != nil {
+		return fmt.Errorf("incorrectly formatted input [%s], err: %v", *element.Value, err)
```

Comment:
minor: I think we should remove the `,  err` part at the end, and just print `... : %s`